docs/crd-changelog.md

```diff
@@ -0,0 +1,138 @@
+# CRD Changelog
+
+This document explains major changes made in new CRD versions. It is intended to help users migrate and take
+advantage of the new features.
+
+## opentelemetry.io/v1beta1 OpenTelemetryCollector
+
+### Migration
+
+There is no need for any user action. The operator will automatically convert existing `v1alpha1` resources to `v1beta1`.
+
+### Structured Configuration
+
+The `Config` field containing the Collector configuration is a string in `v1alpha1`. This has some downsides:
```

Comment:
I would call out the limitation with kustomize and other tools

Reply:
I called it out below, would you prefer it closer to the start of the section?